README.md

```diff
@@ -173,7 +173,7 @@ You will get a high-quality BLAS library (MKL) and you get controlled dependency
 Once you have [Anaconda](https://www.anaconda.com/distribution/#download-section) installed, here are the instructions.
 
 If you want to compile with CUDA support, install
-- [NVIDIA CUDA](https://developer.nvidia.com/cuda-downloads) 10.2 or above
+- [NVIDIA CUDA](https://developer.nvidia.com/cuda-downloads) 10.2 or above - Currently incompatible with 11.5
```

Comment:
this is not correct, master supports 11.5

Reply:
And what is the branch I can edit this for 1.10.2? One can't edit in tags

Reply:
cc @b0nol, I don't know what's the policy towards release branches.

Reply:
We can add it to the release branch, `release/1.10`

Although the central authority for what we do support can be found at https://pytorch.org/get-started/locally/

Reply:
master does support CUDA 11.5 though so this PR isn't actually necessary